draft-ietf-quic-transport.md

```diff
@@ -7334,6 +7334,36 @@ change controller of the IETF and a contact of the QUIC working group
 (quic@ietf.org).
 
 
+## QUIC Versions Registry {#iana-version}
+
+IANA \[SHALL add/has added] a registry for "QUIC Versions" under a "QUIC"
+heading.
+
+The "QUIC Versions" registry governs a 32-bit space; see {{versions}}. This
+registry follows the registration policy from {{iana-policy}}. Permanent
+registrations in this registry are assigned using the Specification Required
+policy ({{!RFC8126}}).
+
+All fields in {{iana-provisional}} MUST be included in all registrations for
+this registry.
+
+The codepoint of 0x00000001 to the protocol is assigned with permanent status
+to the protocol defined in this document. The codepoint of 0x00000000 is
+permanently reserved; the note for this codepoint \[shall] indicate\[s] that
+this version is reserved for Version Negotiation. All codepoints that follow
+the pattern 0x?a?a?a?a are permanently reserved.
```

Comment:
Should this use the formulation from the other greasing sections, that IANA SHALL NOT assign them, rather than actually having them reserved in the list?  I don't want to see 16K registrations for "Reserved" on IANA's list, after all.